odt-opt-in/initial ODT Opt-in JSON Contract.md

```diff
@@ -148,7 +142,6 @@ PowerOn Response if successful
 
 Error Codes (individual error codes are for ease of researching issues.
              All error codes will be returned to the UX as '500')
```

Comment:
I think the 503 for "No eligible shares" should be passed through to the client. We have a specific error screen for that.

Reply:
Pushed new commit with updated error code reporting and to note that all error codes will be reported to UX as "500" with the exception of error code "503"